src/htmx.js

```diff
@@ -1025,7 +1065,7 @@ return (function () {
             return fragment;
         }
 
-        function swap(swapStyle, elt, target, fragment, settleInfo) {
+        function swap(swapStyle, elt, target, fragment, settleInfo, isStandardError) {
```

Comment:
I'm a bit suspicious about passing error flag to `swap` method. Ideally, the decision to use the default value should come from the use site (which would be the `doSwap` method), but I suppose this calls for larger refactoring

Reply:
Yeah that's because the swap function gets `htmx.config.defaultSwapStyle` in the fallback case, and I needed to differentiate from the error counterpart when relevant here
Couldn't find an elegant solution to work around this issue, other than passing that boolean around, which isn't that elegant either I know 😆

Reply:
@Renerick see my [latest commit](https://github.com/bigskysoftware/htmx/pull/1620/commits/665f3de8f2dd580184f83f7345c66d179b5703e9), I prefer that new version, though it still requires passing a parameter down the way, but now it's the default swap style instead of a boolean, let me know what you think about that